src/stories/10_Introduction/_AMP_Story_Hello_World.html

```diff
@@ -0,0 +1,109 @@
+<!-- 
+  ## Introduction
+
+  Learn how to create your first AMP Story. The [amp-story](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md) extension provides a new format for displaying visual content. 
+-->
+<!-- -->
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- AMP Stories are written using AMPHTML and they use their own AMP extension: `amp-story`. The first step is to import the `amp-story` in the header. -->
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+    <!-- AMP Stories can make use of other AMP extensions such as `amp-video`. Howeever, they support only a subset of the available AMP extensions. You can find a full list of the supported extensions [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md#children-of-amp-story-grid-layer). -->
```

Comment:
- fix typo: However
- change "they support" to "AMP Stories support"
- again, link to the reference doc on ampproject.org -> https://www.ampproject.org/docs/reference/components/amp-story

Reply:
done